shark/examples/shark_inference/stable_diffusion/README.md

```diff
@@ -42,3 +42,11 @@ unzip ~/.local/shark_tank/<your unet>/inputs.npz
 
 iree-benchmark-module --module_file=/path/to/output/vmfb --entry_function=forward --function_input=@arr_0.npy --function_input=1xf16 --function_input=@arr_2.npy --function_input=@arr_3.npy --function_input=@arr_4.npy  
 ```
+
+## Use custom checkpoints for a specific version of a model:
+* This feature can only be used with `import_mlir` command-line argument.
+* To try this feature you may download [unet-checkpoint](https://huggingface.co/CompVis/stable-diffusion-v1-4/resolve/main/unet/diffusion_pytorch_model.bin) and provide its path to `--unet_checkpoint` command-line argument.
+```shell
+python main.py --precision=fp32 --device=vulkan --import_mlir --unet_checkpoint=<path to unet's checkpoint file> --prompt="tajmahal, oil on canvas, sunflowers, 4k, uhd"
```

Comment:
doesn't import_mlir avoid downloading and generated the IR via torch-mlir ? @pashu123

Reply:
yes.